test/node_entry_point.ts

```diff
@@ -8,6 +8,7 @@
 
 // Must be loaded before zone loads, so that zone can detect WTF.
 import './wtf_mock';
+import './custom_error';
```

Comment:
it is used here.
https://github.com/JiaLiPassion/zone.js/blob/f318c3b1fce222f0d4551555ca5834b37e4868d6/test/common/Error.spec.ts#L167

which the method is defined here

https://github.com/JiaLiPassion/zone.js/blob/f318c3b1fce222f0d4551555ca5834b37e4868d6/test/custom_error.ts#L13